queries.sql

```diff
@@ -123,4 +123,76 @@ FROM animals
 JOIN owners ON animals.owner_id = owners.id 
 GROUP BY owners.full_name 
 ORDER BY COUNT(*) DESC 
+
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name 
+FROM animals a 
+JOIN visits v ON a.id = v.animal_id 
+JOIN vets vt ON v.vet_id = vt.id 
+WHERE vt.name = 'William Tatcher' 
+ORDER BY v.visit_date DESC 
+LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT v.animal_id) 
+FROM visits v 
+JOIN vets vt ON v.vet_id = vt.id 
+WHERE vt.name = 'Stephanie Mendez';
```

Comment:
- [ ] Stephanie mendez actually visited 4 different animals, but from your screenshot, I can see your query returned 3. THis is due to incorrect data added to the visits table. Kindly fix data in the visits table.
![image](https://github.com/ernestkorir/vet_clinic/assets/94551484/cb3b61cb-0385-406b-93df-e92402a86806)